task_templates/3_pipelines/2_python3_sklearn_regression/create_pipeline.py

```diff
@@ -92,10 +92,14 @@ def to_string(x):
 # For text, we:
 # 1. Impute missing values with the string "missing"
 # 2. Tfidf encode the text, using 1-grams and 2-grams.
+
+# Sklearn's TfidfVectorizer can only handle one column of text at a time,
+# and will fail on datasets with more than one text column.
+# MultiColumnTfidfVectorizer from sagemaker-scikit-learn-extension may be useful to handle multiple text columns.
 text_pipeline = Pipeline(
     steps=[
         ("imputer", SimpleImputer(strategy="constant", fill_value="missing")),
-        ("tfidf", MultiColumnTfidfVectorizer(ngram_range=(1, 2))),
+        ("tfidf", TfidfVectorizer(ngram_range=(1, 2))),
     ]
 )
```

Comment:
please remove all this `text_pipeline` code.  make the comment above more clear with links (see request for changes)